core/src/main/java/org/apache/struts2/interceptor/BackgroundProcess.java

```diff
@@ -75,7 +75,7 @@ public void run() {
      *
      * @throws Exception any exception thrown will be thrown, in turn, by the ExecuteAndWaitInterceptor
      */
-    protected void beforeInvocation() throws Exception {
+    private void beforeInvocation() throws Exception {
```

Comment:
Reducing scope can affect users and blocks adding other implementations.

Reply:
Thanks 👍 . fixed :)